boards/shields/nrf700x_nrf54h20dk/nrf700x_nrf54h20dk.overlay

```diff
@@ -0,0 +1,173 @@
+/* Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+#include <freq.h>
+
+/* Wi-Fi needs more flash (1M) and RAM (512K):
+ *
+ * To minimize the changes, only necessary nodes are added, so, delete
+ * any linked nodes. This was done here as it's applicable to most samples,
+ * if a sample needs anything extra, it can override this with a sample specific
+ * overlay.
+ */
+ /delete-node/ &ipc0;
+ /delete-node/ &cpuppr_vpr;
+ /delete-node/ &mram1x;
+ /delete-node/ &cpurad_uicr;
+
+/ {
+	/delete-node/ ipc;
+	/delete-node/ reserved-memory;
+
+	/* Shift offsets by 512K */
+	soc {
+		mram1x: mram@e000000 {
+			compatible = "nordic,mram";
+			erase-block-size = <4096>;
+			write-block-size = <16>;
+			reg = <0xe000000 DT_SIZE_K(2048)>;
+			cpuapp_rx_partitions: cpuapp-rx-partitions {
+				compatible = "nordic,owned-partitions", "fixed-partitions";
+				status = "okay";
+				perm-read;
+				perm-execute;
+				perm-secure;
+				#address-cells = <1>;
+				#size-cells = <1>;
+
+				cpuapp_slot0_partition: partition@66000 {
+					reg = <0x66000 DT_SIZE_K(1024)>;
+				};
+			};
+
+			cpuapp_rw_partitions: cpuapp-rw-partitions {
+				compatible = "nordic,owned-partitions", "fixed-partitions";
+				status = "okay";
+				perm-read;
+				perm-write;
+				perm-secure;
+				#address-cells = <1>;
+				#size-cells = <1>;
+
+				storage_partition: partition@166000 {
+					reg = <0x166000 DT_SIZE_K(24)>;
+				};
+			};
+		};
+	};
+
+	reserved-memory {
+		compatible = "reserved-memory";
+		#address-cells = < 0x1 >;
+		#size-cells = < 0x1 >;
+		/* Shift offsets by 256K and remove IPC regions */
+		cpuapp_ram0x_region: memory@2f000000 {
+			compatible = "nordic,owned-memory";
+			reg = <0x2f000000 DT_SIZE_K(512)>;
+			status = "okay";
+			perm-read;
+			perm-write;
+			perm-secure;
+			perm-execute;
+			#address-cells = <1>;
+			#size-cells = <1>;
+			ranges = <0x0 0x2f000000 DT_SIZE_K(512)>;
+
+			cpuapp_data: memory@0 {
+				reg = <0x0 DT_SIZE_K(512)>;
+			};
+		};
+
+		/* Below sections are unchanged, duplicated as delete the entire node */
```

Comment:
why is entire-node deletion required?

Reply:
The starting region is the CPU app and as we change the size all other regions have to be adjusted and for Wi-Fi we don't use those regions, so, instead of adjusting the node, deleted the node and only kept the relevant regions. I could delete all unused regions, but if an another region is added in the future, this will break, so, it's better to delete the full node.